src/i2c.rs

```diff
@@ -227,7 +227,7 @@ macro_rules! hal {
                             .start()
                             .start()
                             .autoend()
-                            .software()
+                            .automatic()
```

Comment:
Good catch. Without it the slave on the bus would never know, when to stop writing to master.